install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -107,8 +107,12 @@ spec:
       valueType: STRING
     source.serviceAccount:
       valueType: STRING
+    source.podName:
```

Comment:
@geeknoid how should we add these attributes in your proposed new model?

should we be using `source.workload.name` here instead? 

and, below, should these be: `destination.workload.name` and  `destination.workload.containerName`  ?

Do we care that these are k8s-only attributes?

Reply:
source.podName ends up being source.name in the updated model.
desrtination.podName is destination.name
destination.containerName is destination.container.name